core/modules/catalog/helpers/tax/calculateProductTax.ts

```diff
@@ -0,0 +1,47 @@
+import updateProductPrices from './updateProductPrices'
+import getRate from './getRate'
+
+const calculateProductTax = ({ product, taxClasses, taxCountry = 'PL', taxRegion = '', sourcePriceInclTax = false, deprecatedPriceFieldsSupport = false, userGroupId }) => {
+  const rate = getRate({ taxClasses, product, taxCountry, taxRegion, userGroupId })
+
+  if (rate) {
+    updateProductPrices(product, rate, sourcePriceInclTax, deprecatedPriceFieldsSupport)
```

Comment:
`updateProductPrices(product, rate, sourcePriceInclTax, deprecatedPriceFieldsSupport, currentStoreView().tax.sourcePriceIncludesTax)`

We need to use here the `tax.sourcePriceIncludesTax` from the config value. 

This should probably be be fixed in all versions where the config value is not user for `updateProductPrices`?

Reply:
Yeah, but it's used in the same way as previously: https://github.com/DivanteLtd/vue-storefront/blob/develop/core/modules/catalog/helpers/tax.ts#L245

Reply:
But the way it was previously is just wrong to have it always true as it is true as a default value. 

We need to use the config value there. I can do a PR to fix it. But that is maybe a bug since some versions I have not check that yet.

Reply:
ok so I'll sort it out in this PR as it's related, no problem 👍